servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -93,6 +98,11 @@ public InodeFile build() {
     protected InodeFile.Builder getThis() {
       return this;
     }
+
+    @Override
+    public InodeFile.Builder setPermissionStatus(PermissionStatus ps) {
```

Comment:
Yes, they both have permission. But file and dir have different initial permission. Their difference is defined in `Constants.FILE_DIR_PERMISSION_DIFF`. So for file, this method is override to apply the diff mask.